### PR TITLE
OCP BUGS-2365: Adding support info for oc-mirror

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OCPBUGS-2365

Link to docs preview:
Preview: https://51749--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-support_installing-mirroring-disconnected

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@dmesser it's pretty simple, but does this cover what you were looking for?

Preview: https://51749--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-support_installing-mirroring-disconnected

This PR is only for 4.11 and 4.12 (and I'll remove that note for 4.12). Let me know if we need something in the TP version of the docs for 4.10, but I assume that would be different.

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please QE review this?

Preview: https://51749--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-support_installing-mirroring-disconnected

File: modules/oc-mirror-support.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+:_content-type: CONCEPT
+[id="oc-mirror-support_{context}"]
+= oc-mirror compatibility and support
+
+The oc-mirror plug-in supports mirroring {product-title} payload images and Operator catalogs for {product-title} versions 4.9 and later.
+
+Use the latest available version of the oc-mirror plug-in regardless of which versions of {product-title} you need to mirror.
+
+// TODO: remove this note for 4.12
```
Comment (opayne1) [inline modules/oc-mirror-support.adoc:13]: I was thinking for a minute on this and maybe I am overcomplicating, but would it be easier for you later if you made the above changes in main and then add the important note in a separate PR for 4.11? I guess you could do it either way though.

**Review by opayne1 — COMMENTED** [review]
I just had one technical question. Feel free to disregard if it doesn't apply to this situation. PR content lgtm!

File: modules/oc-mirror-support.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/oc-mirror-support.adoc:13]: Yeah true probably would have been, but I think it'll require 2 PRs anyways so doesn't matter too much

**Pull request merged**